src/modules/projects/components/ProjectOutgoingReferralDetailsSubForm.tsx

```diff
@@ -57,11 +61,24 @@ const ProjectOutgoingReferralDetailsSubForm: React.FC<Props> = ({
     );
   }
 
+  const { data: receivingProject } = useGetCanProjectAcceptReferralQuery({
+    variables: {
+      projectId: destinationProjectId,
+      referrerEnrollmentId: enrollmentId,
+    },
+  });
+
   return (
     <Box>
       {formDefinitionLoading && (
         <LoadingSkeleton width={'100%'} count={1} sx={{ my: 2 }} />
       )}
+      {receivingProject && !receivingProject.canProjectAcceptReferral && (
+        <Alert color='warning' sx={{ mb: 2 }}>
```

Comment:
I noticed this still has a checkbox for the warning. I think if you use severity instead of warning it will adjust the icon as well: https://mui.com/material-ui/api/alert/#alert-prop-severity